src/app/Dashboard/Dashboard.tsx

```diff
@@ -36,19 +36,23 @@
  * SOFTWARE.
  */
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { CardConfig } from '@app/Shared/Redux/Configurations/DashboardConfigSlicer';
 import { dashboardCardConfigDeleteCardIntent, RootState, StateDispatch } from '@app/Shared/Redux/ReduxStore';
 import { FeatureLevel } from '@app/Shared/Services/Settings.service';
 import { TargetView } from '@app/TargetView/TargetView';
-import { Card, CardActions, CardBody, CardHeader, Stack, StackItem, Text } from '@patternfly/react-core';
+import { CardActions, CardBody, CardHeader, Grid, GridItem, gridSpans, Text } from '@patternfly/react-core';
 import * as React from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Observable, of } from 'rxjs';
 import { AddCard } from './AddCard';
 import { AutomatedAnalysisCardDescriptor } from './AutomatedAnalysis/AutomatedAnalysisCard';
 import { DashboardCardActionMenu } from './DashboardCardActionMenu';
+import { ResizableCard } from './ResizableCard';
 
 export interface DashboardCardDescriptor {
   title: string;
+  minimumSpan: gridSpans;
+  defaultSpan: gridSpans;
```

Comment:
Just a thought:

```typescript
interface DashboardCardDescriptor {
  size: {
    span: Sized<gridSpans>
    height: Sized<number>
  }
}

interface Sized<T> {
  minimum: T;
  default: T;
  maximum: T;
}
```

I'm not sure the generic type constraint `T` will work exactly as written, but I think it's illustrative enough.